docs/tutorials/collect-unbound-metrics.md

```diff
@@ -0,0 +1,134 @@
+configuring the collector.
+
+## Configure the Unbound collector
+
+You may not need to do any more configuration to have Netdata collect your Unbound metrics.
+
+If you followed the steps above to enable `remote-control` and make your Unbound files readable by Netdata, that should
+be enough. Restart Netdata with `service netdata restart`, or the appropriate method for your system. You should see
+Unbound metrics in your Netdata dashboard!
+
+If that failed, you will need to manually configure `unbound.conf`. See the next section for details.
+
+### Manual setup for a local Unbound server
+
+To configure Netdata's Unbound collector module, navigate to your Netdata configuration directory (typically at
+`/etc/netdata/`) and use `edit-config` to initialize and edit your Unbound configuration file.
+
+```bash
+cd /etc/netdata/ # Replace with your Netdata configuration directory, if not /etc/netdata/
+sudo ./edit-config go.d/unbound.conf
+```
+
+The file contains all the global and job-related parameters. The `name` setting is required, and two Unbound servers
+can't have the same name.
+
+> It is important you know whether your Unbound server is running in cumulative or non-cumulative mode, as a conflict
+> between modes will create incorrect charts.
+
+Here are two examples for local Unbound servers, which may work based on your unique setup:
+
+```yaml
+jobs:
+  - name: local
+    address: 127.0.0.1:8953
+    cumulative: no
+    use_tls: yes
+    tls_skip_verify: yes
+    tls_cert: /path/to/unbound_control.pem
+    tls_key: /path/to/unbound_control.key
+  
+  - name: local
+    address: 127.0.0.1:8953
+    cumulative: yes
+    use_tls: no
+```
+
+Netdata will attempt to read `unbound.conf` to get the appropriate `address`, `cumulative`, `use_tls`, `tls_cert`, and
+`tls_key` parameters. 
+
+Restart Netdata with `service netdata restart`, or the appropriate method for your system.
+
+### Manual setup for a remote Unbound server
+
+Collecting metrics from remote Unbound servers requires manual configuration. There are too many possibilities to cover
+all remote connections here, but the [default `unbound.conf`
+file](https://github.com/netdata/go.d.plugin/blob/master/config/go.d/unbound.conf) contains a few useful examples:
+
+```yaml
+jobs:
+  - name: remote
+    address: 203.0.113.10:8953
+    use_tls: no
+
+  - name: remote_cumulative
+    address: 203.0.113.11:8953
+    use_tls: no
+    cumulative: yes
+
+  - name: remote
+    address: 203.0.113.10:8953
+    cumulative: yes
+    use_tls: yes
+    tls_cert: /etc/unbound/unbound_control.pem
+    tls_key: /etc/unbound/unbound_control.key
+```
+
+To see all the available options, see the default [unbound.conf
+file](https://github.com/netdata/go.d.plugin/blob/master/config/go.d/unbound.conf).
+
+## Tweak Unbound collector alarms
+
+**COMING SOON: Ilya is adding/updating alarms and I will finish this section then.**
```

Comment:
No alarms today. We decided to add them later. I created the issue btw

https://github.com/NLnetLabs/unbound/issues/123

Reply:
Got it. We can update this tutorial after they're up.